spinalcordtoolbox/reports/assets/index.html

```diff
@@ -68,10 +68,20 @@
               <div id="dropdown" class="dropdown">
                   <button id="dropdownButton" class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true">Display Columns <span class="caret"></span></button>
                   <ul class="dropdown-menu">
-                      <li><a class="dropdown-item" id="Date" onclick="toggleColumn(id)">Display Date &#10004</button></a></li>
-                      <li><a class="dropdown-item" id="Dataset" onclick="toggleColumn(id)">Display Dataset &#10004</a></li>
-                      <li><a class="dropdown-item" id="Subject" onclick="toggleColumn(id)">Display Subject &#10004</a></li>
-                      <li><a class="dropdown-item" id="Function" onclick="toggleColumn(id)">Display Function &#10004</a></li>
+                      <li><a class="dropdown-item" id="Date" onclick="toggleColumn(id)">Display Date</button></a></li>
```

Comment:
a lot of these fields are used internally. Just stick to the following (x means default):
- Date: x
- Dataset: 
- Subject: x
- File (rename from Input_file_name): x
- Contrast
- Function: x

Reply:
> UPDATE 2019-06-07 11.01:
You can remove the "Display" in the dropdown menu (people get it)

Remove the "display" from the button, the internal elements of the menu or both?

Reply:
just from the internal elements